src/include/common/statement.h

```diff
@@ -87,7 +84,6 @@ class Statement {
 
   // cached plan tree
   std::shared_ptr<planner::AbstractPlan> plan_tree;
```

Comment:
I checked the code. Don't know why this isn't a unique_ptr. All of its uses are like this: "statement->GetPlanTree().get()".
Your call if you wanna change it.